package.json

```diff
@@ -4,6 +4,9 @@
   "author": "Takuya Asano <takuya.a@gmail.com>",
   "description": "JavaScript implementation of Japanese morphological analyzer",
   "main": "./dist/node/kuromoji.js",
+  "browser": {
+    "./dist/node/loader/NodeDictionaryLoader.js": "./dist/node/loader/BrowserDictionaryLoader.js"
```

Comment:
I don't know about `browser` field in details, but if this line is replaced by
```
"./src/loader/NodeDictionaryLoader.js": "./src/loader/BrowserDictionaryLoader.js"
```
, perhaps we don't need this line of code in gulpfile.js:
```
b.require(__dirname + "/src/loader/BrowserDictionaryLoader.js", {expose: "./loader/NodeDictionaryLoader.js"});
```
(I think browserify will make this rearrangement)